operations/agent.go

```diff
@@ -185,6 +186,15 @@ func Agent() cli.Command {
 			ctx, cancel := context.WithCancel(context.Background())
 			defer cancel()
 
+			h, err := host.FindOneId(ctx, opts.HostID)
```

Comment:
Can we do this here? I'm not familiar with the agent command, but I don't think it has a database connection to get a host- so I think this would stop agents from starting

Reply:
ah yeah, this is my bad
made a new agent method for this